options/rtdl/generic/linker.hpp

```diff
@@ -61,7 +61,7 @@ struct ObjectRepository {
 
 	void _discoverDependencies(SharedObject *object, uint64_t rts);
 
-	frg::hash_map<frg::string_view, SharedObject *,
+	frg::hash_map<frg::string<MemoryAllocator>, SharedObject *,
```

Comment:
Using a full allocated string shouldn't be necessary here if the string view is over the allocated string in `SharedObject`.